dataprofiler/tests/profilers/profiler_options/test_unstructured_text_profile_options.py

```diff
@@ -0,0 +1,138 @@
+from dataprofiler.profilers.profiler_options import TextProfilerOptions, \
+    BooleanOption
+from dataprofiler.tests.profilers.profiler_options.test_base_inspector_options \
+     import TestBaseInspectorOptions
```

Comment:
is this proper?

Reply:
can you elaborate?

Reply:
import being on the next line, I'm not sure what python standards suggest.